### PR TITLE
What's section updated.

### PR DESCRIPTION
**Pull request by UditNS**
I have changes other section also little bit so that it will utilize full width in small devices. If you want any changes to made then comment or mail me.

### CODE REVIEW

**Comment by aryamankha** [comment]
On small screens, the "Latest drops" section is bleeding into the FAQs and the images on "What's it all about" seem to have the wrong aspect ratio.

**Comment by UditNS** [comment]
It may caused by other sections, I have 2 commits one include only what section responsiveness but in that case the width was not full used so I made another commit which includes other section little bit changes so that what's section will use full width.

I have also attached the pictures of what's section in this issue comment where we talked earlier in that it is fine.

You just need to have other section to be responsive.

File: components/drops.tsx [hunk 1]
```diff
@@ -4,9 +4,9 @@ export default function Drops() {
         <div className="drops-section bg-white h-max px-5 py-10">
             <div className="header bg-white">
                 <h1 className="text-black text-4xl font-bold font-sans ml-5">Latest Drops</h1>
-                <div className="bg-purple-500 h-1 mt-2 w-96 ml-5"></div>
+                <div className="bg-purple-500 h-1 mt-2 w-64 md:w-96 ml-5"></div>
             </div>
-            <div className="card-container flex flex-row gap-5 h-96">
+            <div className="card-container flex md:flex-row flex-col gap-5 h-96">
```
Comment (aryamankha) [inline components/drops.tsx:9]: Making this section flex-col on small screens is causing the overflow issue - you can see how on small screens this is causing a column structure which affects the layout.

**Comment by aryamankha** [comment]
I added a comment showing which change you made that's causing the issue. I think you tried to make drops.tsx responsive too, but it created a different UI bug

File: components/drops.tsx (hunk 1 above)
Comment (UditNS) [reply components/drops.tsx:9]: Then what should I do make, if I make it row then this look little different and rows are not good in small devices

**Comment by UditNS** [comment]
Screenshot_2022_1013_231530

Before any changes website looks like this

**Comment by aryamankha** [comment]
Okay - I'll merge for now, and then open another issue for the other sections

**Comment by aryamankha** [comment]
Were you able to use Vivid? Would love to hear your thoughts!

**Comment by UditNS** [comment]
Sorry @aryamankha  for this issue.

I can try for vivid but I don't know much about it.

**Comment by UditNS** [comment]
Something like this

**Pull request merged**